packages/semcom-demo-app/src/demo.component.ts

```diff
@@ -0,0 +1,156 @@
+import { html, css, CSSResult, TemplateResult, state, PropertyValues, query, unsafeCSS } from 'lit-element';
+import { createMachine, interpret, State } from 'xstate';
+import { from } from 'rxjs';
+import { RxLitElement } from 'rx-lit';
+import { map } from 'rxjs/operators';
+import { Purpose, Source, AuthenticateComponent, LoadingComponent, ProviderListComponent, ProviderListItemComponent, SolidSDKService } from '@digita-ai/ui-transfer-components';
+import { ComponentMetadata } from '@digita-ai/semcom-core';
+import { Theme } from '@digita-ai/ui-transfer-theme';
```

Comment:
I'm not entirely convinced of using UI Transfer's theme here, but we'll just try it for a while.